client/package.json

```diff
@@ -35,23 +33,21 @@
     "react-dom": "^15.3.1",
     "react-router": "^4.1.1",
     "react-router-dom": "^4.1.1",
-    "reactify": "^1.1.1",
-    "socket.io-client": "^1.3.6",
-    "vinyl-source-stream": "^1.1.0"
+    "socket.io-client": "^1.3.6"
   },
   "devDependencies": {
+    "babel-core": "^6.26.0",
     "babel-jest": "^20.0.3",
+    "babel-loader": "^7.1.2",
```

Comment:
Don't you also need a jsx loader? I get the following error when trying to transpile:

```
dionyziz@erdos ~/workspace/ting/client (webpack*) % webpack
Hash: ee5ca700647e16f1b9f1
Version: webpack 1.13.2
Time: 77ms
      Asset     Size  Chunks             Chunk Names
    main.js  1.69 kB       0  [emitted]  main
main.js.map   1.6 kB       0  [emitted]  main
   [0] ./main.js 22 bytes {0} [built] [1 error]
   [1] ./app.jsx 0 bytes [built] [failed]

ERROR in ./app.jsx
Module parse failed: /Users/dionyziz/workspace/ting/client/app.jsx Unexpected token (40:12)
You may need an appropriate loader to handle this file type.
SyntaxError: Unexpected token (40:12)
    at Parser.pp$4.raise (/usr/local/lib/node_modules/webpack/node_modules/acorn/dist/acorn.js:2221:15)
    at Parser.pp.unexpected (/usr/local/lib/node_modules/webpack/node_modules/acorn/dist/acorn.js:603:10)
    at Parser.pp$3.parseExprAtom (/usr/local/lib/node_modules/webpack/node_modules/acorn/dist/acorn.js:1822:12)
    at Parser.pp$3.parseExprSubscripts (/usr/local/lib/node_modules/webpack/node_modules/acorn/dist/acorn.js:1715:21)
    at Parser.pp$3.parseMaybeUnary (/usr/local/lib/node_modules/webpack/node_modules/acorn/dist/acorn.js:1692:19)
    at Parser.pp$3.parseExprOps (/usr/local/lib/node_modules/webpack/node_modules/acorn/dist/acorn.js:1637:21)
    at Parser.pp$3.parseMaybeConditional (/usr/local/lib/node_modules/webpack/node_modules/acorn/dist/acorn.js:1620:21)
    at Parser.pp$3.parseMaybeAssign (/usr/local/lib/node_modules/webpack/node_modules/acorn/dist/acorn.js:1597:21)
    at Parser.pp$3.parseParenAndDistinguishExpression (/usr/local/lib/node_modules/webpack/node_modules/acorn/dist/acorn.js:1861:32)
    at Parser.pp$3.parseExprAtom (/usr/local/lib/node_modules/webpack/node_modules/acorn/dist/acorn.js:1796:19)
    at Parser.pp$3.parseExprSubscripts (/usr/local/lib/node_modules/webpack/node_modules/acorn/dist/acorn.js:1715:21)
    at Parser.pp$3.parseMaybeUnary (/usr/local/lib/node_modules/webpack/node_modules/acorn/dist/acorn.js:1692:19)
    at Parser.pp$3.parseExprOps (/usr/local/lib/node_modules/webpack/node_modules/acorn/dist/acorn.js:1637:21)
    at Parser.pp$3.parseMaybeConditional (/usr/local/lib/node_modules/webpack/node_modules/acorn/dist/acorn.js:1620:21)
    at Parser.pp$3.parseMaybeAssign (/usr/local/lib/node_modules/webpack/node_modules/acorn/dist/acorn.js:1597:21)
    at Parser.pp$3.parseExpression (/usr/local/lib/node_modules/webpack/node_modules/acorn/dist/acorn.js:1573:21)
    at Parser.pp$1.parseReturnStatement (/usr/local/lib/node_modules/webpack/node_modules/acorn/dist/acorn.js:839:33)
    at Parser.pp$1.parseStatement (/usr/local/lib/node_modules/webpack/node_modules/acorn/dist/acorn.js:699:34)
    at Parser.pp$1.parseBlock (/usr/local/lib/node_modules/webpack/node_modules/acorn/dist/acorn.js:981:25)
    at Parser.pp$3.parseFunctionBody (/usr/local/lib/node_modules/webpack/node_modules/acorn/dist/acorn.js:2105:24)
    at Parser.pp$3.parseMethod (/usr/local/lib/node_modules/webpack/node_modules/acorn/dist/acorn.js:2075:10)
    at Parser.pp$3.parsePropertyValue (/usr/local/lib/node_modules/webpack/node_modules/acorn/dist/acorn.js:2004:25)
    at Parser.pp$3.parseObj (/usr/local/lib/node_modules/webpack/node_modules/acorn/dist/acorn.js:1989:14)
    at Parser.pp$3.parseExprAtom (/usr/local/lib/node_modules/webpack/node_modules/acorn/dist/acorn.js:1805:19)
    at Parser.pp$3.parseExprSubscripts (/usr/local/lib/node_modules/webpack/node_modules/acorn/dist/acorn.js:1715:21)
    at Parser.pp$3.parseMaybeUnary (/usr/local/lib/node_modules/webpack/node_modules/acorn/dist/acorn.js:1692:19)
    at Parser.pp$3.parseExprOps (/usr/local/lib/node_modules/webpack/node_modules/acorn/dist/acorn.js:1637:21)
    at Parser.pp$3.parseMaybeConditional (/usr/local/lib/node_modules/webpack/node_modules/acorn/dist/acorn.js:1620:21)
    at Parser.pp$3.parseMaybeAssign (/usr/local/lib/node_modules/webpack/node_modules/acorn/dist/acorn.js:1597:21)
    at Parser.pp$3.parseExprList (/usr/local/lib/node_modules/webpack/node_modules/acorn/dist/acorn.js:2165:22)
 @ ./main.js 1:0-20
```

Reply:
`babel-preset-react` is in the dependencies already, and `babel-loader` relies on `.babelrc` which includes the react preset (I didn't change it). Are you sure you have everything installed correctly, including `devDependencies`?

Reply:
You're right, it needed an `rm -rf node_modules/ && npm install`.